pkgs/by-name/js/jsoncons/package.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
```

I think the `finalAttrs` pattern should be used here, it allows for more effective overriding: https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes
```suggestion
stdenv.mkDerivation (finalAttrs: {
```